week02/valid_anagram.cpp

```diff
@@ -0,0 +1,19 @@
+class Solution {
+public:
+    long long int mod = 10007;
+    unsigned long long getHash(string &str){
+        unsigned long long hashValue = 1;
+        for(int i = 0; i < str.size();i++){
+            hashValue = (hashValue * (257 + (str[i] - 'a')))% mod;
+        }
+        return hashValue;
+    }
```

Comment:
```suggestion
    long long mod = 202206214218227LL;
    long long getHash(string &str){
        long long hashValue = 1;
        for(auto c : str){
            hashValue = (hashValue * (257 + c)) % mod;
        }
        return hashValue;
    }
```
It has better coverage.